2017-07-21 17:59:31,228 - FileController.py - INFO: 读取yaml文件：../TestCase/Config/config.yaml
2017-07-21 17:59:31,232 - FileController.py - INFO: 创建yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-21 17:59:31,232 - CaseGoto.py - INFO: 临时文件成功： D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-21 17:59:31,232 - CaseGoto.py - INFO: Header 配置文件中未配置，可能引起出错
2017-07-21 17:59:31,240 - FileController.py - INFO: 读取excel文件：D:\GitPro\Python\PythonApiTest\TestCase\TestData\Qone.xlsx
2017-07-21 17:59:31,240 - CaseGoto.py - INFO: CaseList: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.1', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': '', 'Depends': '', 'Response_Type': 'Json', 'Checkpoint': {'RESPONSE_STATUS': 'OK', 'ERROR_MESSAGE': None, 'NAME': '魏双双'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-21 17:59:31,241 - test_ApiTest_2.py - INFO: caseindextest_api_rq_1
2017-07-21 17:59:31,241 - CaseInteg.py - INFO: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.1', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': '', 'Depends': '', 'Response_Type': 'Json', 'Checkpoint': {'RESPONSE_STATUS': 'OK', 'ERROR_MESSAGE': None, 'NAME': '魏双双'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-21 17:59:31,242 - DataGenerate.py - ERROR: 获取参数值失败,可能含有二次包装数据,请再次调用 {'APP_KEY': '10001', 'DEVICE_VERSION': '112', 'FORMAT': 'json', 'TELPHONE': '15502168672', 'TIMESTAMP': '2017-07-21 17:59:31', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'zaq123', 'Sign': '$sign$'}
2017-07-21 17:59:31,242 - CaseInteg.py - INFO: 请求表单已处理
2017-07-21 17:59:31,242 - CaseInteg.py - INFO: {'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '112', 'FORMAT': 'json', 'TELPHONE': '15502168672', 'TIMESTAMP': '2017-07-21 17:59:31', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'zaq123', 'Sign': '$sign$'}}
2017-07-21 17:59:31,242 - UserParam.py - INFO: B4B83DC748B8E511A2846C0B840997C3APP_KEY10001DEVICE_VERSION112FORMATjsonPASSWORDzaq123TELPHONE15502168672TIMESTAMP2017-07-21 17:59:31U_IDANDROIDVERSION1.0B4B83DC748B8E511A2846C0B840997C3
2017-07-21 17:59:31,242 - DataGenerate.py - INFO: {'APP_KEY': '10001', 'DEVICE_VERSION': '112', 'FORMAT': 'json', 'TELPHONE': '15502168672', 'TIMESTAMP': '2017-07-21 17:59:31', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'zaq123', 'Sign': '78986915D3DF66352A2CA8FDD04A796F'}
2017-07-21 17:59:31,242 - RequestGenerate.py - INFO: {'url': 'https://sitqone-wh.quarkfinance.com/api/CobraUser/Login', 'method': 'post', 'headers': '', 'json': {'APP_KEY': '10001', 'DEVICE_VERSION': '112', 'FORMAT': 'json', 'TELPHONE': '15502168672', 'TIMESTAMP': '2017-07-21 17:59:31', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'zaq123', 'Sign': '78986915D3DF66352A2CA8FDD04A796F'}}
2017-07-21 17:59:31,242 - CaseInteg.py - INFO: 请求数据已解析
2017-07-21 17:59:31,570 - RequestGenerate.py - INFO: <Response [200]>
2017-07-21 17:59:31,570 - CaseInteg.py - INFO: 请求数据已发送
2017-07-21 17:59:31,570 - CaseInteg.py - INFO: 请求返回数据：<Response [200]>
2017-07-21 17:59:31,570 - test_ApiTest_2.py - INFO: 用例文件中未配置需要收集的参数，若有后续依赖，执行会报错
2017-07-21 17:59:31,570 - test_ApiTest_2.py - INFO: 返回值Json字符串：{'BODY': '{"RESULT":true,"MESSAGE":"登录成功","MODEL":{"ID":1196,"NAME":"魏双双","TELPHONE":"15502168672","CREATED_TIME":"2017-06-19 09:53:38","CITY":"QuarkFinance","ID_NO":"41152119910617391X","STAFF_NO":"2016050301","IS_SET_GESTURE_PASSWORD":true,"IS_SIGN_IN":false,"PHONE_OS":0,"DEPARTMENT_NAME":"技术部","USER_ACCOUNT_DEPARTMENT_ID":"83","USER_COMPANY_ID":"323ea269-5aea-4f60-b6d8-31d3215e208d","USER_ACCOUNT":"shuangshuangwei","IS_ACTIVED":1,"AGREEMENT_ON_CONFIDENTIALITY":"Y"}}', 'RESPONSE_STATUS': 'OK', 'ELAPSE_TIME': '', 'ERROR_CODE': None, 'ERROR_MESSAGE': None}
2017-07-21 17:59:31,571 - Utils.py - INFO: NAME  in  登录成功  遍历结束
2017-07-21 17:59:31,571 - Utils.py - INFO: NAME  in  OK  遍历结束
2017-07-21 17:59:31,571 - Utils.py - INFO: NAME  in    遍历结束
2017-07-21 17:59:31,571 - ResultGenerate.py - INFO: CasePath:D:\GitPro\Python\PythonApiTest\TestCase\TestData\Qone.xlsx_case	CaseNumb:No.1	CaseName:登录	Request_Url:/api/CobraUser/Login	Temp_Filepath:D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml	CaseIndex:test_api_rq_1	caseresult:Pass	
2017-07-21 17:59:31,571 - FileController.py - INFO: 写入yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-21 17:59:31,573 - test_ApiTest_2.py - INFO: {'test_api_rq_1': '/api/CobraUser/Login,D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case,No.1,登录,Pass'}
2017-07-21 17:59:31,573 - test_ApiTest_2.py - INFO: 返回值已处理
